xml/System.Threading/Barrier.xml

```diff
@@ -930,7 +930,8 @@
         <param name="millisecondsTimeout">The number of milliseconds to wait, or <see cref="F:System.Threading.Timeout.Infinite" />(-1) to wait indefinitely.</param>
         <param name="cancellationToken">The <see cref="T:System.Threading.CancellationToken" /> to observe.</param>
         <summary>Signals that a participant has reached the barrier and waits for all other participants to reach the barrier as well, using a 32-bit signed integer to measure the timeout, while observing a cancellation token.</summary>
-        <returns>if all participants reached the barrier within the specified time; otherwise false</returns>
+        <returns>
+          <see langword="true" /> if all participants reached the barrier within the specified time; otherwise <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if all participants reached the barrier within the specified time; otherwise, <see langword="false" />.</returns>
```